transport_plugins/websocket/iotile_transport_websocket/device_adapter.py

```diff
@@ -1,28 +1,47 @@
+import logging
+from builtins import range
 from iotile.core.hw.transport.adapter import DeviceAdapter
 from iotile.core.utilities.validating_wsclient import ValidatingWSClient
-from iotile.core.exceptions import ArgumentError, HardwareError, TimeoutExpiredError
+from iotile.core.hw.reports.parser import IOTileReportParser
+from iotile.core.exceptions import ArgumentError, HardwareError, ValidationError
 from connection_manager import ConnectionManager
+import messages
 
 
 class WebSocketDeviceAdapter(DeviceAdapter):
-    """ A device adapter allowing connections to devices over websockets
+    """ A device adapter allowing connections to devices over WebSockets
 
     Args:
-        url (string): A url for the websocket server in form of server:port/path
+        port (string): A url for the WebSocket server in form of server:port
     """
 
     def __init__(self, port):
         super(WebSocketDeviceAdapter, self).__init__()
 
+        # Configuration
         self.set_config('default_timeout', 10.0)
         self.set_config('expiration_time', 60.0)
         self.set_config('probe_supported', True)
         self.set_config('probe_required', True)
+        self.mtu = int(self.get_config('mtu', 60*1024))  # Split script payloads larger than this
 
-        path = "ws://{0}/services".format(port)
+        # Set logger
+        self.logger = logging.getLogger('ws.manager')
```

Comment:
Standard logging practice is to use the name of the module rather than a specific name so that it's easy for a consume to know how to turn loggers on and off by name:

```
self.logger = logging.getLogger(__name__)
```

The code in `bled112` transport plugin is old and contains this bad practice as well.  Here's a good reference on python logging best practices:
https://fangpenlin.com/posts/2012/08/26/good-logging-practice-in-python/